packages/quantic/force-app/main/default/lwc/quanticSearchBox/quanticSearchBox.js

```diff
@@ -120,31 +151,38 @@ export default class QuanticSearchBox extends LightningElement {
     );
   }
 
+  get searchBoxValue() {
+    return this.searchBox?.state.value || '';
+  }
```

Comment:
we no longer want to manage the value in the `quanticSearchBoxInput`? Why pass it as props?

Reply:
who said we no longer want to manage it x) We do want to manage the value in quanticSearchBoxInput, and that's why it's passed a prop as the value in  the `quanticSearchBoxInput` needs always to match the value we have in the state.

i just simplified how we manage this value.